### PR TITLE
Updated Security faq of TIDB Cloud FAQ

### PR DESCRIPTION
**Pull request by lulukelu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/secure-connections-to-serverless-tier-clusters.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ TiDB Serverless Tier allows only TLS connections and rejects non-TLS connections
 
 ## What TLS versions can I use?
 
-TiDB Serverless Tier only supports TLS 1.2 and TLS 1.3.
+TiDB Serverless Tier supports TLS 1.2 and TLS 1.3.
```
Comment (qiancai) [inline tidb-cloud/secure-connections-to-serverless-tier-clusters.md:18]:
```suggestion
TiDB Serverless Tier supports TLS 1.2 and TLS 1.3.
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 2]
```diff
@@ -119,16 +119,44 @@ No. TiFlash data cannot be exported.
...
-No. TiDB Cloud runs on the PingCAP VPC, but the data and traffic are encrypted by default. So you do not need to worry about data privacy issues.
+TiDB Cloud uses storage volume encryption by default for your business data at rest,including your database data and backups data. TiDB Cloud requires TLS encryption for your business data in transit and using component TLS encryption your business data in your Database Cluster between TiDB,PD ,TiKV and Tiflash.
+
+If your organization requires more specific business data encryption information in TiDB Cloud,please contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
+
+### Can TiDB Cloud run in your VPC?
+
+No. TiDB Cloud is Database as a service(DBaaS) and runs only on the TiDB Cloud VPC, which is a cloud computing managed service offering that provides access to a database without requiring the setup of physical hardware, the installation of software.
+
+### How secure is your TiDB Cluster?
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:136]:
```suggestion
### Is my TiDB cluster secure?
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 3]
```diff
@@ -119,16 +119,44 @@ No. TiFlash data cannot be exported.
...
+
+In TiDB Cloud, we offer dedicated Tier and Serverless Tier for different customers.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:138]:
```suggestion
In TiDB Cloud, you can use either a Dedicated Tier cluster or a Serverless Tier cluster according to your needs.
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 4]
```diff
@@ -119,16 +119,44 @@ No. TiFlash data cannot be exported.
...
+
+In dedicated Cluster, we created independent sub-accounts and VPCs for each cluster, set up firewall rules to isolate external connections, create server-side TLS certificates and component TLS certificates for each cluster to encrypted cluster data in transit, and provide IP access rules for each cluster to protect that only allowed source IPs can access your cluster.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:140]:
```suggestion
For Dedicated Tier clusters, TiDB Cloud ensures cluster security with the following measures:

- Creates independent sub-accounts and VPCs for each cluster.
- Sets up firewall rules to isolate external connections.
- Creates server-side TLS certificates and component-level TLS certificates for each cluster to encrypt cluster data in transit.
- Provide IP access rules for each cluster to ensure that only allowed source IPs can access your cluster.
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 5]
```diff
@@ -119,16 +119,66 @@ No. TiFlash data cannot be exported.
 
 ## Security FAQs
 
-### How does TiDB protect data privacy and ensure security?
+### Is TiDB Cloud secure?
```
Comment (ljun0712) [inline tidb-cloud/tidb-cloud-faq.md:122]: 1、这个问题太大了，建议具体到某一个点。
2、这里的回答仅仅是关于数据加密的内容。建议修改问题描述，比如："How does TiDB Cloud encrypt my data?"

File: tidb-cloud/tidb-cloud-faq.md [hunk 6]
```diff
@@ -119,16 +119,66 @@ No. TiFlash data cannot be exported.
...
 
-Transport Layer Security (TLS) and Transparent Data Encryption (TDE) are included for encryption at rest. There are two different network planes: the application to the TiDB server and the plane for data communication. We include extended syntax to compare Subject Alternative Name for verification of certificates and TLS context for internal communication.
+In TiDB Cloud, all data at rest is encrypted, and all network traffic is encrypted using Transport Layer Security(TLS). 
 
-### Can TiDB Cloud run in our VPC?
+- Encryption of data at rest is automated using encrypted storage volumes. 
+- Encryption of data in transit between your client and your cluster is automated using TiDB Cloud web server TLS and TiDB cluster TLS.
 
-No. TiDB Cloud runs on the PingCAP VPC, but the data and traffic are encrypted by default. So you do not need to worry about data privacy issues.
+### How does TiDB Cloud encrypt your business data?
```
Comment (ljun0712) [inline tidb-cloud/tidb-cloud-faq.md:129]: 1、这个问题也是关于数据加密，似乎与上一个问题重复，考虑合并一下？
2、问题统一站在用户视角，问题中的人称都使用"my"、"our"类，不使用"you"、"your"

File: tidb-cloud/tidb-cloud-faq.md [hunk 7]
```diff
@@ -119,16 +119,66 @@ No. TiFlash data cannot be exported.
...
+- Creates server-side TLS certificates and component-level TLS certificates for each cluster to encrypt cluster data in transit.
+- Provide IP access rules for each cluster to ensure that only allowed source IPs can access your cluster.
+
+For Serverless Tier clusters, TiDB Cloud ensures cluster security with the following measures:
+
+- Creates independent sub-accounts for each cluster.
+- Sets up firewall rules to isolate external connections.
+- Provides cluster server TLS certificates to encrypt cluster data in transit.
+
+All connections to the cluster over the internet use TLS 1.2 or 1.3.
```
Comment (ljun0712) [inline tidb-cloud/tidb-cloud-faq.md:156]: 关于支持TLS版本，可以单独起一个问题

File: tidb-cloud/tidb-cloud-faq.md [hunk 8]
```diff
@@ -119,16 +119,66 @@ No. TiFlash data cannot be exported.
...
+### How do I connect to my database in a TiDB cluster?
+
+For a Dedicated Tier cluster, the steps to connect your cluster are simplified as follows:
+
+1. Authorize your network.
+2. Set up your database users and login credentials.
+3. Download and configure TLS for your cluster server.
+4. Choose a SQL client and execute an auto-generated command displayed on the TiDB Cloud UI, and then connect to your cluster through the SQL client.
+
+For detailed steps, see [Connect to Your TiDB Cluster](/tidb-cloud/connect-to-tidb-cluster.md).
```
Comment (ljun0712) [inline tidb-cloud/tidb-cloud-faq.md:167]: 保持格式一致，采用以下两种方式处理：
1、此处的更多详细信息的提示，放到步骤4结尾，不单独起一行
2、回答完Dedicated Tier和Serverless Tier之后，单独起一行给出“详细信息的提示”

File: tidb-cloud/tidb-cloud-faq.md [hunk 9]
```diff
@@ -119,16 +119,66 @@ No. TiFlash data cannot be exported.
...
+
+For a Serverless Tier cluster, the steps to connect your cluster are simplified as follows: 
+
+1. Set a database user and login credentials. 
+2. Choose a SQL client, execute an auto-generated command displayed on the TiDB Cloud UI, and then connect to your cluster easily. For more information, see [Connect to Your TiDB Cluster](/tidb-cloud/connect-to-tidb-cluster.md).
+
+### Who has access to my business data of a database cluster?
```
Comment (ljun0712) [inline tidb-cloud/tidb-cloud-faq.md:174]: 这个问题回答的是TiDB Cloud Support是否能访问数据库集群的业务数据，不是讲解的谁有权限访问。
1、建议修改问题，关注点在"TiDB Cloud Support不能访问数据库业务数据"
2、回答中"TiDB Cloud Support"，作为一个整体，Support首字母应该大写，或者单独标记出来，阅读起来更流畅

File: tidb-cloud/tidb-cloud-faq.md (hunk 6 above)
Comment (qiancai) [reply tidb-cloud/tidb-cloud-faq.md:129]: +1

File: tidb-cloud/tidb-cloud-faq.md (hunk 8 above)
Comment (qiancai) [reply tidb-cloud/tidb-cloud-faq.md:167]: +1

File: tidb-cloud/tidb-cloud-faq.md (hunk 9 above)
Comment (qiancai) [reply tidb-cloud/tidb-cloud-faq.md:174]: +

File: tidb-cloud/tidb-cloud-faq.md [hunk 10]
```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
...
+In TiDB Cloud, all data at rest is encrypted, and all network traffic is encrypted using Transport Layer Security(TLS). 
 
-### Can TiDB Cloud run in our VPC?
+- Encryption of data at rest is automated using encrypted storage volumes. 
+- Encryption of data in transit between your client and your cluster is automated using TiDB Cloud web server TLS and TiDB cluster TLS.
 
-No. TiDB Cloud runs on the PingCAP VPC, but the data and traffic are encrypted by default. So you do not need to worry about data privacy issues.
+### How does TiDB Cloud encrypt my business data?
+
+TiDB Cloud uses storage volume encryption by default for your business data at rest, including your database data and backup data. TiDB Cloud requires TLS encryption for your business data in transit and uses component-level TLS encryption for your business data in your database cluster between TiDB, PD, TiKV, and TiFlash.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:131]:
```suggestion
TiDB Cloud uses storage volume encryption by default for your business data at rest, including your database data and backup data. TiDB Cloud requires TLS encryption for data in transit, and also requires component-level TLS encryption for data in your database cluster between TiDB, PD, TiKV, and TiFlash.
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 11]
```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
...
+
+If you require more specific information about business data encryption in TiDB Cloud, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
+
+### What versions of TLS does TiDB Cloud support?
+
+All connections to the cluster over the internet use TLS 1.2 or 1.3.
+
+### Can I run TiDB Cloud in my VPC?
+
+No. TiDB Cloud is Database-as-a-Service (DBaaS) and runs only on the TiDB Cloud VPC. As a cloud computing managed service, TiDB Cloud provides access to a database without requiring the setup of physical hardware and the installation of software.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:141]:
```suggestion
No. TiDB Cloud is Database-as-a-Service (DBaaS) and runs only in the TiDB Cloud VPC. As a cloud computing managed service, TiDB Cloud provides access to a database without requiring the setup of physical hardware and the installation of software.
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 12]
```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
...
+### Is my TiDB cluster secure?
+
+In TiDB Cloud, you can use either a Dedicated Tier cluster or a Serverless Tier cluster according to your needs.
+
+For Dedicated Tier clusters, TiDB Cloud ensures cluster security with the following measures:
+
+- Creates independent sub-accounts and VPCs for each cluster.
+- Sets up firewall rules to isolate external connections.
+- Creates server-side TLS certificates and component-level TLS certificates for each cluster to encrypt cluster data in transit.
+- Provide IP access rules for each cluster to ensure that only allowed source IPs can access your cluster.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:152]:
```suggestion
- Provide IP access rules for each cluster to ensure that only allowed source IP addresses can access your cluster.
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 13]
```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
...
+
+For Serverless Tier clusters, TiDB Cloud ensures cluster security with the following measures:
+
+- Creates independent sub-accounts for each cluster.
+- Sets up firewall rules to isolate external connections.
+- Provides cluster server TLS certificates to encrypt cluster data in transit.
+
+### How do I connect to my database in a TiDB cluster?
+
+For a Dedicated Tier cluster, the steps to connect your cluster are simplified as follows:
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:162]:
```suggestion
For a Dedicated Tier cluster, the steps to connect to your cluster are simplified as follows:
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 14]
```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
...
+
+1. Authorize your network.
+2. Set up your database users and login credentials.
+3. Download and configure TLS for your cluster server.
+4. Choose a SQL client and execute an auto-generated command displayed on the TiDB Cloud UI, and then connect to your cluster through the SQL client.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:167]:
```suggestion
4. Choose a SQL client, get an auto-generated connection string displayed on the TiDB Cloud UI, and then connect to your cluster through the SQL client using the string.
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 15]
```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
...
+
+For a Serverless Tier cluster, the steps to connect your cluster are simplified as follows: 
+
+1. Set a database user and login credentials. 
+2. Choose a SQL client, execute an auto-generated command displayed on the TiDB Cloud UI, and then connect to your cluster easily.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:172]:
```suggestion
2. Choose a SQL client, get an auto-generated connection string displayed on the TiDB Cloud UI, and then connect to your cluster through the SQL client using the string.
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 16]
```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
...
+
+For more information, see [Connect to Your TiDB Cluster](/tidb-cloud/connect-to-tidb-cluster.md).
+
+### Who has access to my business data of a database cluster?
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:176]:
```suggestion
### Can TiDB Cloud Support access the business data in my TiDB cluster?
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 17]
```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
...
+
+TiDB Cloud Support cannot directly access the data in your database cluster. The only exception is that when you need to improve products and solve cluster operation problems, TiDB Cloud Support can access the cluster operation data after you provide your internal temporary authorization. All authorization and access records are audited annually by third-party audit organizations. For example, PCI-DSS, SOC2, and ISO27701.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:178]:
```suggestion
TiDB Cloud Support cannot directly access the data in your TiDB cluster. The only exception is that when you need to improve products and solve cluster operation problems, TiDB Cloud Support can access the cluster operation data after you provide your internal temporary authorization. All authorization and access records are audited annually by third-party audit organizations. For example, PCI-DSS, SOC2, and ISO27701. 
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 18]
```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
...
+### How do I connect to my database in a TiDB cluster?
+
+For a Dedicated Tier cluster, the steps to connect your cluster are simplified as follows:
+
+1. Authorize your network.
+2. Set up your database users and login credentials.
+3. Download and configure TLS for your cluster server.
+4. Choose a SQL client and execute an auto-generated command displayed on the TiDB Cloud UI, and then connect to your cluster through the SQL client.
+
+For a Serverless Tier cluster, the steps to connect your cluster are simplified as follows:
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:169]:
```suggestion
For a Serverless Tier cluster, the steps to connect to your cluster are simplified as follows: 
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 19]
```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
...
+For a Dedicated Tier cluster, the steps to connect your cluster are simplified as follows:
+
+1. Authorize your network.
+2. Set up your database users and login credentials.
+3. Download and configure TLS for your cluster server.
+4. Choose a SQL client and execute an auto-generated command displayed on the TiDB Cloud UI, and then connect to your cluster through the SQL client.
+
+For a Serverless Tier cluster, the steps to connect your cluster are simplified as follows: 
+
+1. Set a database user and login credentials.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:171]:
```suggestion
1. Set a database user and login credential. 
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 20]
```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
...
+
+No. TiDB Cloud is Database-as-a-Service (DBaaS) and runs only in the TiDB Cloud VPC. As a cloud computing managed service, TiDB Cloud provides access to a database without requiring the setup of physical hardware and the installation of software.
+
+### Is my TiDB cluster secure?
+
+In TiDB Cloud, you can use either a Dedicated Tier cluster or a Serverless Tier cluster according to your needs.
+
+For Dedicated Tier clusters, TiDB Cloud ensures cluster security with the following measures:
+
+- Creates independent sub-accounts and VPCs for each cluster.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-faq.md:149]: sub-account or sub-accounts for each cluster?

File: tidb-cloud/tidb-cloud-faq.md [hunk 21]
```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
...
+For a Serverless Tier cluster, the steps to connect to your cluster are simplified as follows: 
+
+1. Set a database user and login credential. 
+2. Choose a SQL client, get an auto-generated connection string displayed on the TiDB Cloud UI, and then connect to your cluster through the SQL client using the string.
+
+For more information, see [Connect to Your TiDB Cluster](/tidb-cloud/connect-to-tidb-cluster.md).
+
+### Who has access to my business data of a database cluster?
+
+TiDB Cloud Support cannot directly access the data in your TiDB cluster. The only exception is that when you need to improve products and solve cluster operation problems, TiDB Cloud Support can access the cluster operation data after you provide your internal temporary authorization. All authorization and access records are audited annually by third-party audit organizations. For example, PCI-DSS, SOC2, and ISO27701.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-faq.md:178]:
```suggestion
TiDB Cloud Support cannot directly access the data in your TiDB cluster. The only exception is that when you need to improve products and solve cluster operation problems, TiDB Cloud Support can access the cluster operation data after you provide your internal temporary authorization. All authorization and access records are audited annually by third-party audit organizations, for example, PCI-DSS, SOC2, and ISO27701. 
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 22]
```diff
@@ -119,13 +119,65 @@ No. TiFlash data cannot be exported.
...
 ## Security FAQs
 
-### How does TiDB protect data privacy and ensure security?
+### Is TiDB Cloud secure?
 
-Transport Layer Security (TLS) and Transparent Data Encryption (TDE) are included for encryption at rest. There are two different network planes: the application to the TiDB server and the plane for data communication. We include extended syntax to compare Subject Alternative Name for verification of certificates and TLS context for internal communication.
+In TiDB Cloud, all data at rest is encrypted, and all network traffic is encrypted using Transport Layer Security(TLS). 
 
-### Can TiDB Cloud run in our VPC?
+- Encryption of data at rest is automated using encrypted storage volumes.
```
[the remark this answers is not in the document]
Comment (lulukelu) [reply tidb-cloud/tidb-cloud-faq.md:126]: yes ， auto-encrypted using default encryption key

[the remark this answers is not in the document]
Comment (lulukelu) [reply tidb-cloud/tidb-cloud-faq.md:126]: auto-encrypted using encrypted storage volumes

File: tidb-cloud/tidb-cloud-faq.md (hunk 20 above)
Comment (lulukelu) [reply tidb-cloud/tidb-cloud-faq.md:149]: it is better to change sub-account into customer account

Comment (lulukelu) [reply tidb-cloud/tidb-cloud-faq.md:149]: agree to delete this sentences

File: tidb-cloud/tidb-cloud-faq.md [hunk 23]
```diff
@@ -119,16 +119,68 @@ No. TiFlash data cannot be exported.
...
+
+TiDB Cloud operational data is described in [TiDB Cloud Privacy Policy](https://www.pingcap.com/privacy-policy/) and [TiDB Cloud Data Processing Agreement](https://www.pingcap.com/legal/data-processing-agreement-for-tidb-cloud-services/).
 
 ## Support FAQ
 
 ### What support is available for customers?
 
-TiDB Cloud is supported by the same team behind TiDB, which has run mission-critical use cases for over 1500 global enterprises across industries including financial services, e-commerce, enterprise applications, and gaming. TiDB Cloud offers a free basic support plan for each user and you can upgrade to a paid plan for extended services. For more information, see [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
+ TiDB Cloud is supported by the same team behind TiDB, which has run mission-critical use cases for over 1500 global enterprises across industries including financial services, e-commerce, enterprise applications, and gaming. TiDB Cloud offers a free basic support plan for each user and you can upgrade to a paid plan for extended services. For more information, see [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:186]:
```suggestion
TiDB Cloud is supported by the same team behind TiDB, which has run mission-critical use cases for over 1500 global enterprises across industries including financial services, e-commerce, enterprise applications, and gaming. TiDB Cloud offers a free basic support plan for each user and you can upgrade to a paid plan for extended services. For more information, see [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 24]
```diff
@@ -119,16 +119,68 @@ No. TiFlash data cannot be exported.
...
+For a Serverless Tier cluster, the steps to connect to your cluster are simplified as follows: 
+
+1. Set a database user and login credential. 
+2. Choose a SQL client, get an auto-generated connection string displayed on the TiDB Cloud UI, and then connect to your cluster through the SQL client using the string.
+
+For more information, see [Connect to Your TiDB Cluster](/tidb-cloud/connect-to-tidb-cluster.md).
+
+### Who has access to my business data of a database cluster?
+
+Only customers can access own table data in their own TiDB cluster.TiDB Cloud Support cannot directly access the data in your TiDB cluster. The only exception is that when you need to improve products and solve cluster operation problems, TiDB Cloud Support can access the cluster operation data after you provide your internal temporary authorization. All authorization and access records are audited annually by third-party audit organizations, for example, PCI-DSS, SOC2, and ISO27701.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:178]:
```suggestion
Only you can access your table data in your own TiDB cluster. TiDB Cloud Support cannot directly access the data in your TiDB cluster. The only exception is that when you need to improve products and solve cluster operation problems, TiDB Cloud Support can access the cluster operation data after you provide your internal temporary authorization. All authorization and access records are audited annually by third-party audit organizations, for example, PCI-DSS, SOC2, and ISO27701. 
```

**Review by qiancai — APPROVED** [review]

**Review by ljun0712 — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**